src/index.js

```diff
@@ -186,11 +186,11 @@ function shimModel(target) {
 
       const loaders = options[EXPECTED_OPTIONS_KEY].loaders;
       let loader = loaders[this.name].byPrimaryKey;
-      if (options.raw) {
-        const cacheKey = getCacheKey(this, this.primaryKeyAttribute, {raw: options.raw});
+      if (options.raw || options.paranoid === false) {
```

Comment:
is `true` the default?

Reply:
I think it's not the default because you have to opt into a model being paranoid and using the magical deletedAt column

Reply:
I meant more if it's the default when paranoid is enabled.
`options.paranoid = false` means don't fetch paranoid records right? Or does it mean, do fetch paranoid records?

Reply:
when `false` is provided, soft-deleted rows will be returned 

https://github.com/sequelize/sequelize/blob/b0d4f06ea8a089a569f18f53b5958aa0cf82e671/test/integration/model/paranoid.test.js#L86-L89